<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Pet</class>
 <widget class="QWidget" name="Pet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QTextEdit{
	border:1px solid rgb(0,0,0);
	border-radius:5px;
	background-color: rgb(255, 255, 255);
	color: rgb(0, 0, 0);
}
QScrollBar:vertical{
	border: none;
	width:8px;
	margin: 0 0 0 0;
	background: transparent;
}
QScrollBar::handle:vertical{
	background: rgb(200, 200, 200);
	border-radius:4px;
}
QScrollBar::handle:vertical:hover{
	background: rgb(160, 160, 160);
	border-radius:4px;
}
QScrollBar::handle:vertical:pressed{
	background: rgb(5, 160, 255);
	border-radius:4px;
}
QScrollBar::add-page,QScrollBar::sub-page:vertical{
	background: transparent;
}
QScrollBar::add-line,QScrollBar::sub-line:vertical{
	background: transparent;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,5,1,1">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTextEdit" name="textEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Ignored">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'楷体'; font-size:24pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'楷体';&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="showWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Ignored">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="timelabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Ignored">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>2019.12.17-2023.12.31</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="timelabel_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Ignored">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
